django_cloud_deploy/cli/new.py

```diff
@@ -118,13 +124,17 @@ def main(args: argparse.Namespace):
     if remaining_parameters_to_prompt:
```

Comment:
This has the an issue if they want to use an exisiting project but they pass in all the arguments. The check to confirm that the name is correct wouldn't happen.